datastore/tests/integration/conftest.py

```diff
@@ -16,4 +16,8 @@ def kind() -> str:
 
 @pytest.fixture(scope='module')  # type: ignore
 def project() -> str:
-    return 'voiceai-staging'
+    return 'fststaging'
+
+@pytest.fixture(scope='module')  # type: ignore
+def export_bucket_name() -> str:
+    return os.environ['DATASTORE_EXPORT_BUCKET_NAME']
```

Comment:
You can set this to `voiceai-staging-public-test` (or a folder within it)

Reply:
i did this as a os enviorn so by default the test doesn't run since it can take 10s of seconds.